src/openforms/js/components/form/file.js

```diff
@@ -18,12 +19,37 @@ const REGISTRATION = {
       key: 'registration.informatieobjecttype',
       label: 'Informatieobjecttype',
       tooltip: 'Save the attachment in the Documents API with this InformatieObjectType.',
-      dataSrc: 'url',
+      dataSrc: 'custom',
       data: {
-        // if the url starts with '/', then formio will prefix it with the formio
-        // base URL, which is of course wrong. We there explicitly use the detected
-        // host.
-        url: getFullyQualifiedUrl('/api/v2/registration/plugins/zgw/informatieobjecttypen'),
+        custom(context) {
+          const instance = context.instance;
+          const registrationInfo =
+            instance?.options?.openForms?.registrationBackendInfoRef?.current;
+          if (!registrationInfo) return [];
+
+          // if the url starts with '/', then formio will prefix it with the formio
+          // base URL, which is of course wrong. We there explicitly use the detected
+          // host.
+          let url = new URL(getFullyQualifiedUrl('/api/v2/registration/informatieobjecttypen'));
```

Comment:
```suggestion
          const url = '/api/v2/registration/informatieobjecttypen';
```
we don't need this workaround anymore because there's no formio config sitting in the way anymore (so you can also drop the comment above). I think you can then also just use the literal string in the `get` call instead of assigning to an intermediate variable.